pyvo/dal/sia2.py

```diff
@@ -0,0 +1,1064 @@
+        an observation to the user. In queries it is generally better to refer
+        to astronomical objects by position, using a name resolver to convert
+        the target name into a coordinate (when possible).
+        """
+        return self['target_name'].decode('utf-8')
+
+    @property
+    def target_class(self):
+        """
+        This field indicates the type of object that was pointed for this
+        observation. It is a string with possible values defined in a special
+        vocabulary set to be defined: list of object classes (or types) used
+        by the SIMBAD database, NED or defined in another IVOA vocabulary.
+        """
+        if 'target_class' in self.keys():
+            return self['target_class'].decode('utf-8')
+        return None
+
+    ###          DATA DESCRIPTION
+    @property
+    def id(self):
+        """
+        Collection specific nternal ID given by the ObsTAP service
+        """
+        return self['obs_id'].decode('utf-8')
+
+    @property
+    def title(self):
+        """
+        Brief description of dataset in free format
+        """
+        if 'obs_title' in self.keys():
+            return self['obs_title'].decode('utf-8')
+        return None
+
+    @property
+    def collection(self):
+        """
+        The name of the collection (DataID.Collection) identifies the data
+        collection to which the data product belongs. A data collection can be
+        any collection of datasets which are alike in some fashion. Typical
+        data collections might be all the data from a particular telescope,
+        instrument, or survey. The value is either the registered shortname
+        for the data collection, the full registered IVOA identifier for the
+        collection, or a data provider defined short name for the collection.
+        Examples: HST/WFPC2, VLT/FORS2, CHANDRA/ACIS-S, etc.
+        """
+        return self['obs_collection'].decode('utf-8')
+
+    @property
+    def create_date(self):
+        """
+        Date when the dataset was created
+        """
+        if 'obs_create_date' in self.keys():
+            return dateutil.parser.isoparse(self['obs_create_date'])
+        return None
+
+    @property
+    def creator_name(self):
+        """
+        The name of the institution or entity which created the dataset.
+        """
+        if 'obs_creator_name' in self.keys():
+            return self['obs_creator_name'].decode('utf-8')
+        return None
+
+    @property
+    def creator_did(self):
+        """
+        IVOA dataset identifier given by its creator.
+        """
+        if 'obs_creator_did' in self.keys():
+            return self['obs_creator_did'].decode('utf-8')
+        return None
+
+    ##         CURATION INFORMATION
+    @property
+    def release_date(self):
+        """
+        Observation release date
+        """
+        if 'obs_release_date' in self.keys():
+            return time.Time(self['obs_release_date'])
```

Comment:
Astropy Time?

Reply:
Not a good idea? i thought it conveniently takes care of parsing...

Reply:
Mea culpa, thought i saw time.time somewhere above.